ajax/libs/jquery.quicksearch/package.json

```diff
@@ -11,27 +11,30 @@
     },
     {
       "name": "Rik Lomas",
-      "url": "http://www.riklomas.co.uk/"
+      "url": "https://www.riklomas.co.uk/"
     }
   ],
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/DeuxHuitHuit/quicksearch.git"
+    "url": "https://github.com/DeuxHuitHuit/quicksearch.git"
```

Comment:
I think url as the same with https://github.com/DeuxHuitHuit/quicksearch/blob/master/package.json#L15 it fine.
How do you like the url?

Thanks.